lib/remote/configpackageutility.hpp

```diff
@@ -59,7 +62,8 @@ class ConfigPackageUtility
 	static void WritePackageConfig(const String& packageName);
 	static void WriteStageConfig(const String& packageName, const String& stageName);
 
-	static void TryActivateStageCallback(const ProcessResult& pr, const String& packageName, const String& stageName, bool activate, bool reload);
+	static void TryActivateStageCallback(const ProcessResult& pr, const String& packageName, const String& stageName, bool activate,
+		bool reload, const Shared<Defer>::Ptr& resetPackageUpdates = nullptr);
```

Comment:
What's the default for?

Reply:
Not sure, does someone want to be able to use this method without this param or not? Did not want to break anything.

Reply:
But the uses where the parameter is used are the only ones, aren't they? So doesn't really hurt to have the default, but it also isn't needed though.